liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -413,6 +413,15 @@ protected void doExecute() throws Exception {
             return;
         }
         
+        // look for a .sRunning file to check if the server has already started
+        if (serverDirectory.exists()) {
+            File sRunning = new File(serverDirectory.getCanonicalPath()  + "/workarea/.sRunning");
+            if (sRunning.exists()) {
+                log.error("The server " + serverName + " is already running. Terminate all instances of the server before starting dev mode.");
+                System.exit(0);
```

Comment:
Do we really need to call System.exit(0) or just return should be enough

Reply:
If we use `System.exit(0)` the output looks like this:
![image](https://user-images.githubusercontent.com/26146482/63169794-b412a100-c005-11e9-801e-f8a2d3e48194.png)

Alternatively, if we use `return` the output looks like this: 
![image](https://user-images.githubusercontent.com/26146482/63169821-c096f980-c005-11e9-9071-5cd256bcc487.png)

Should we still have the `Build Success` output show even if dev mode has been blocked?

Reply:
If you want to fail the build, you should throw a MojoExecutionException with your error message.

Reply:
Changed! The new error message now looks like this: 
![image](https://user-images.githubusercontent.com/26146482/63171561-cabaf700-c009-11e9-8134-39687a1fd1ed.png)